mungegithub/mungers/cherrypick-label-unapproved.go

```diff
@@ -69,6 +69,11 @@ func (LabelUnapprovedPicks) Munge(obj *github.MungeObject) {
 		return
 	}
 
+	if _, err := obj.GetPR(); err != nil {
+		glog.Errorf("Failed to retrieve PR: %s", err)
```

Comment:
GetPR should have already `glog`'d   as should `ListFiles`.  However I think this is wrong. The problem is the `IsForBranch` not having an error return...   Seems like me getting https://github.com/kubernetes/contrib/pull/1357 merged would be a better fix...